tools/perf-automation/Azure.Sdk.Tools.PerfAutomation/tests.yml

```diff
@@ -102,6 +102,28 @@ Services:
       PackageVersions:
       - azure-storage-blobs: source
 
+- Service: core
+  Languages:
+    Java:
+      Project: sdk/storage/azure-storage-perf
+      PrimaryPackage: azure-core
+      PackageVersions: &java-storage-package-versions
+      - azure-storage-blob: 12.16.1
+        azure-storage-file-share: 12.12.2
+        azure-storage-file-datalake: 12.9.1
+        azure-core: 1.28.0
+        azure-core-http-netty: 1.12.0
+        reactor-core: 3.4.17
+      - azure-storage-blob: 12.16.1
+        azure-storage-file-share: 12.12.2
+        azure-storage-file-datalake: 12.9.1
+        azure-core: source
+        azure-core-http-netty: source
```

Comment:
This is a special argument?

Reply:
yeah, it tells the tool to "not override" value in pom. So effectively value in pom must be "current" to work properly.
see.
https://github.com/Azure/azure-sdk-tools/blob/393813462d44b4db47fe46c66fe941a66b65e5f1/tools/perf-automation/Azure.Sdk.Tools.PerfAutomation/Java.cs#L59-L69